examples/python/2_large_model/compare_load_results.py

```
@@ -0,0 +1,82 @@
+
+    print("=" * 65)
+    print("Model Loading Time Comparison: DashInfer vs SGLang")
+    print("=" * 65)
+    print(f"{'Model:':<20} {di.get('model_path', 'N/A')}")
+    print(f"{'DashInfer GPUs:':<20} {di.get('gpus', 'N/A')}")
+    print(f"{'SGLang TP:':<20} {sg.get('tp', 'N/A')}")
+    print(f"{'Data type:':<20} {di.get('data_type', 'N/A')} / {sg.get('dtype', 'N/A')}")
+    print()
+
+    # DashInfer phases
+    print(f"DashInfer (direct-load):")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)